frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -1,86 +1,85 @@
-import React, { Component } from 'react';
-
+import React, { useState } from 'react';
+import { createRecipe } from '../../lib/apiClient';
 import Footer from '../Footer/Footer';
 import './EditRecipeForm.css';
 
-class EditRecipeForm extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      title: '', // eslint-disable-line react/no-unused-state
-      snippet: '', // eslint-disable-line react/no-unused-state
-    };
-    this.handleInputChange = this.handleInputChange.bind(this);
-  }
+const EditRecipeForm = ({ user }) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [difficulty, setDifficulty] = useState('Advanced');
+  const [duration, setDuration] = useState('60 mins');
```

Comment:
I believe duration is an integer on the backend, so this will need to be converted before being submitted.